### ISSUE TITLE
incorrect basionym author for synonym Arceuthobium laricis

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1957]
[Originally posted on GoogleCode (id 2091) on 2014-01-31]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/6571

<b>What data are incorrect or missing?</b>
Arceuthobium laricis (Piper) H. St.John

<b>What data are you expecting instead?</b>
Arceuthobium laricis (M.E.Johes) H. St.John
Piper made the combination: Razoumofskya laricis (M.E.Jones) Piper
basionym: Arceuthobium douglasii var. laricis M.E.Jones

<b>If applicable, please provide an authoritative source.</b>
MBG, IPNI

**Comment by sjmeades** [comment]
[Originally posted on GoogleCode on 2014-01-31 16:48Z]

Also, Arceuthobium cyanocarpum (A. Nelson) Abrams

should be Arceuthobium cyanocarpum (A. Nelson) J.M.Coulter & A.Nelson 1909, non Abrams 1940

re: IPNI, MBG

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2014-01-31 18:43Z]

The errors come from the FNA draft, which had not been nomenclaturally revised before being circulated for regional review, of course.

Thanks: I will also point out the issue to the taxon editor (who will transmit to the author). Even if it were caught by Gandhi, I would not have seen this until the vol. is published...

Added the further synonyms based on the same basionym.

the cyanocarpum authorities are for the nom. inval. that is syn with the taxon anyway... the author missed this - I kept the original with nom. inval. and added the correct name + syns

Luc
